plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/session/Session.kt

```diff
@@ -39,7 +40,7 @@ class Session(val tabID: String, val project: Project) {
     var isAuthenticating: Boolean
 
     init {
-        context = FeatureDevSessionContext(project)
+        context = FeatureDevSessionContext(project, maxProjectSizeBytes)
```

Comment:
question: why adding the size to the context object as an input?

You could just add the value as a private val inside the class and use it.

Reply:
Maybe another team want to use this class with different `maxProjectSizeBytes`

Reply:
I would doubt so, this is a specific class from our feature, very improbable that it will be a shared class.

Reply:
> very improbable that it will be a shared class.

At least this class now not under FeatureDev folder, so it's already in a shared space, even no one is using it

Reply:
I think that class should be renamed at some point. It is being used by anothe team and is not located in /featureDev folder.

Reply:
I see the file has also a TODO that also shows it is an intended shared functionality